sormas-api/src/main/java/de/symeda/sormas/api/symptoms/SymptomsDto.java

```diff
@@ -201,463 +210,477 @@ public static SymptomsDto build() {
 		return symptoms;
 	}
 
-	@Diseases({ Disease.AFP, Disease.EVD, Disease.GUINEA_WORM, Disease.LASSA, Disease.NEW_INFLUENZA, Disease.CHOLERA, Disease.YELLOW_FEVER,
-			Disease.DENGUE, Disease.PLAGUE, Disease.POLIO, Disease.ANTHRAX, Disease.CORONAVIRUS, Disease.UNSPECIFIED_VHF, Disease.UNDEFINED,
-			Disease.OTHER })
+	@Diseases({AFP, EVD, GUINEA_WORM, LASSA, NEW_INFLUENZA, CHOLERA, YELLOW_FEVER,
```

Comment:
Thanks for highly improving the readability of this :-)

Reply:
👍